test-main.js

```diff
@@ -12,7 +12,7 @@ System.config({
   defaultJSExtensions: true,
   paths: {
     'angular2/*': 'node_modules/angular2/*.js',
-    'rx': 'node_modules/angular2/node_modules/rx/dist/rx.js'
+    '@reactivex/rxjs/*': 'node_modules/@reactivex/rxjs/*.js',
```

Comment:
It depends on the version of npm. In npm 3.x.x is used flat module structure, which makes the test dependent on that, I guess.